spec/features/form/check_answers_page_spec.rb

```diff
@@ -220,13 +220,13 @@
       it "they can click a button to skip sections until the next incomplete section" do
         visit("/logs/#{skip_section_case_log.id}/household-characteristics/check-answers")
         click_link("Save and go to next incomplete section")
-        expect(page).to have_current_path("/logs/#{skip_section_case_log.id}/tenancy-code")
+        expect(page).to have_current_path("/logs/#{skip_section_case_log.id}/property-information/check-answers")
```

Comment:
Is this one still testing what it was supposed to?

Reply:
Previously we had both tenancy code and tenant code in the test form.
I deleted the tenancy code section and kept the tenant code (now renamed tenancy code).

Reply:
Yep, but doesn't that now mean this test doesn't have the data it expected? (And incomplete question left to navigate to)?

Reply:
It is hard for me to assess tbh. These tests were written with a specific configuration in mind, the problem is that our configuration drives database fields. I am not sure we are getting much value from these tests, either they should check a general logic (based on hardcoded states) or test an existing configuration used in production.

Reply:
I think they _are_ intended to check a general logic (the test form fixtures are just arbitrary after all and not meant to reflect a real form (although you're right that it's somewhat constrained by the database coupling)). So we could have just left the old question/section in there (and just changed names to something else if confusing).

 Think the purpose of *this* particular test was to check that the "Save and go to next incomplete section" button works correctly for the scenario where there is an incomplete section still to go (which with the form changes is no longer the case here).
 
 Let me know if that does/doesn't make sense, might be easier to pair or chat through.

Reply:
the purpose of this test is to check that the save and go to next incomplete section button would actually skip a completed subsection. E.g. for the real form if you were on the household characteristics subsection and had already completed the household needs subsection (the next one in the form) then clicking the button would skip that one and take you to the household situation subsection

Reply:
Hope the above clarifies the intended behaviour but if not I'm also happy to pair on this

Reply:
So pairing on this, the test is still correct.

The confusion comes from the fact that the next section is already prefilled to some extent ( in-progress case log which prefills the postcode). When that happens, we go to the check answers page of that section rather than the first question.